text/0000-buildpack-uris.md

```diff
@@ -0,0 +1,128 @@
+# Meta
+[meta]: #meta
+- Name: Buildpack URIs
+- Start Date: 02/07/2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: "N/A"
+
+# Summary
+[summary]: #summary
+
+A set of predefined URIs for referring to buildpacks from various sources (protocols). 
+
+_Please note that the scheme denotes protocol and not type._
+
+# Motivation
+[motivation]: #motivation
+
+Given that we support a multitude of sources for buildpacks, a certain level of ambiguity has naturally presented
+itself. We currently _try_ to resolve the various sources through a complex set of rules yet the ambiguity to users
+still exists. In config files, this ambiguity is at times resolved by introducing a separate key name to signify
+the source (protocol).
+
+The aim of this RFC is to propose a set of URI schemes that could be used in both CLI arguments and config files
+while solving the ambiguity problem.
+
+A few references to further express the problem:
+
+- [A poll](https://buildpacks.slack.com/archives/C94UJCNV6/p1580920669081100) to understand what users expect when
```

Comment:
This is great! can we put the results somewhere durable?

Reply:
We *could* pay for our slack instance :) somehow. But for now, here's an image of the poll: 
<img width="797" alt="Screen Shot 2020-02-12 at 1 39 10 PM" src="https://user-images.githubusercontent.com/5925347/74365981-1e31b600-4d9d-11ea-849e-8cb0ca7b06c0.png">